.github/workflows/docker.yml

```diff
@@ -37,20 +50,18 @@ jobs:
         with:
           token: ${{ secrets.METANORMA_CI_PAT_TOKEN }}
 
-      - name: Test individual templates
-        shell: bash
-        run: |
-          source doctypes.env
-          for template in $DOCTYPES
-          do
-            metanorma new test/$template --type $TYPE --doctype $template --template . --overwrite
-            if [ -f test/${template}/document.adoc ]
-            then
-              metanorma compile --no-install-fonts test/${template}/document.adoc
-            else
-              for docfile in test/${template}/*.adoc
-              do
-                metanorma compile --no-install-fonts $docfile
-              done
-            fi
-          done
+      - uses: metanorma/metanorma-build-scripts/docker-gem-install@master
+
+      - uses: actions-mn/cli/new@main
+        with:
+          type: ${{ matrix.type }}
+          doctype: ${{ matrix.doctype }}
+          output-path: test/${{ matrix.doctype }}
+          template: '.'
+          overwrite: true
+
+      - uses: actions-mn/cli/site-gen@main
```

Comment:
@camobap this needs to changed to run a single file, not `site generate` because this is a single file.

Reply:
Got you, initially I thought that metanorma new generated correct `metanorma.yml` manifest, but looks like it doesn't.

BTW when do we have manifest in this repo at all?

Reply:
We have a `metanorma.yml` file for the whole repo:
https://github.com/metanorma/mn-templates-ogc/blob/master/metanorma.yml

> Got you, initially I thought that metanorma new generated correct metanorma.yml manifest, but looks like it doesn't.

Good point. Can you add an issue in metanorma/metanorma repo so that `metanorma new` will provide a `metanorma.yml` file? Thanks.

Reply:
Actually if we provide the `metanorma.yml` file after template generation then we don't need to change this PR. Can you help with this @CAMOBAP ? Thanks.